stdlib/lexing.mli

```diff
@@ -187,7 +187,7 @@ val flush_input : lexbuf -> unit
 
 (**/**)
 
-(** {1  } *)
+(** {1 Internal functions} *)
```

Comment:
I would propose to delete this section to be consistent with the changes in `parser.mli`.